src/showcase/src/pages/404.astro

```diff
@@ -0,0 +1,10 @@
+---
+import "$src/styles/main.css";
+---
+
+<div id="pageContent"></div>
```

Comment:
you can create a component for `<div id="pageContent"></div>` and `import "$src/styles/main.css";` (it's repeated three times in this PR)?

Reply:
I really hate this `pageContent` thingy so I wanted to avoid hiding it in a component/layout. I'll leave it like this for now and see what the future brings...